src/connectors/cisco_umbrella.py

```diff
@@ -0,0 +1,227 @@
+"""Cisco Umbrella
+Collect Cisco Umbrella information using a Client ID and Secret
+"""
+
+from runners.helpers import db, log
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+from datetime import datetime
+
+import requests
+from .utils import yaml_dump
+
+PAGE_SIZE = 500
+UMBRELLA_URL_DEVICE_BY_PAGE = "https://management.api.umbrella.com/v1/organizations/ORGANIZATION_ID/roamingcomputers"
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'api_key',
+        'title': "Cisco Umbrella",
+        'prompt': "Your Cisco Umbrella API Key",
+        'type': 'str',
+        'required': True,
+        'secret': True,
+    },
+    {
+        'name': 'api_secret',
+        'title': "Cisco Umbrella API Secret",
+        'prompt': "Your Cisco Umbrella API Secret",
+        'type': 'str',
+        'secret': True,
+        'required': True,
+    },
+    {
+        'name': 'organizations_id',
+        'title': "Cisco Umbrella Organizations Id",
+        'prompt': "Your Cisco Umbrella Organizations Id",
+        'type': 'str',
+        'secret': True,
+        'required': True,
+    },
+]
+
+LANDING_TABLE_COLUMNS = [
+    ('DEVICE_ID', 'VARCHAR(256)'),
+    ('OS_VERSION_NAME', 'VARCHAR(256)'),
+    ('LAST_SYNC_STATUS', 'VARCHAR(256)'),
+    ('TYPE', 'VARCHAR(256)'),
+    ('VERSION', 'VARCHAR(256)'),
+    ('LAST_SYNC', 'VARCHAR(256)'),
+    ('OS_VERSION', 'VARCHAR(256)'),
+    ('NAME', 'VARCHAR(256)'),
+    ('STATUS', 'VARCHAR(256)'),
+    ('ORIGIN_ID', 'NUMBER(38,0)'),
+    ('APPLIED_BUNDLE', 'NUMBER(38,0)'),
+    ('HAS_IP_BLOCKING', 'NUMBER(38,0)'),
+]
+
+
+def get_col_transform(idx: int) -> str:
```

Comment:
Ideally the get_col_transform and related code would be integrated into db.py and then connectors use it from there, so that we're not duplicating code across multiple connectors. I think Eduardo is working on that?